orca/src/kani.rs

```diff
@@ -330,11 +330,11 @@ impl KaniLdapServer {
         admin_pw: String,
         ldap_uri: String,
         basedn: String,
-    ) -> Result<Self, ()> {
+    ) -> Result<Box<Self>, ()> {
```

Comment:
Why do we need `Box`?

Reply:
Clippy! Clippy warned that this struct was getting "large" so it prefered to box the content.